ci/build_docs.sh

```diff
@@ -13,17 +17,19 @@ export RAPIDS_VERSION_NUMBER="$RAPIDS_VERSION_MAJOR_MINOR"
 rapids-dependency-file-generator \
   --output conda \
   --file-key docs \
-  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}" | tee env.yaml
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}" \
+  --prepend-channel "${CPP_CHANNEL}" \
+  --prepend-channel "${PYTHON_CHANNEL}" \
+  --prepend-channel conda-forge \
+  --prepend-channel nvidia \
+  --prepend-channel "${DGL_CHANNEL}" \
```

Comment:
`DGL_CHANNEL` is set further down in the script, in that block starting

```shell
if [[ "${RAPIDS_CUDA_VERSION}" == "11.8.0" ]]; then
```

That all needs to be pulled up above this for that to work.